conwayste/src/client.rs

```diff
@@ -960,6 +979,15 @@ impl MainState {
             return Ok(());
         }
 
+        if keycode == KeyCode::Key1 {
```

Comment:
Should move these into the match underneath

```rust
KeyCode::Key1 => {
    ...
}
k if k >= KeyCode::Key2 && k <= KeyCode::Key0 => {
    ...
}
KeyCode::Return => {
    ...
}
...
```

Reply:
I agree. I'll move this.